src/main/java/frc/robot/subsystems/vision/Result.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class Result {
```

Comment:
What is the Java version you use? If it's 14+ (it should be) then you can use [`Record`](https://www.baeldung.com/java-record-keyword) here.